buildspec.yml

```diff
@@ -0,0 +1,17 @@
+version: 0.2
+
+phases:
+  build:
+    commands:
+      - make test-artifacts-in-docker
+  post_build:
+    commands:
+      - cp out/test-artifacts/*.exe .
+artifacts:
+  files:
+      - 'agent.exe'
```

Comment:
I think instead of doing `agent.exe` and `hostsetup.ps1` like this, can you just use the [zip file](https://github.com/aws/amazon-ecs-agent/blob/v1.16.1/scripts/stage#L74-L91) (the logic will probably need to be extracted from there, and then you can do it in `post_build`)?

Reply:
I was letting codebuild do the zip logic for me (this preference is stored in the project settings rather than the buildspec).

I'm inclined to continue letting codebuild zip, but that's only because I don't want to write/copy code.

Reply:
Well, the point of what I'm saying is I'd like the artifact we use for testing to better match what we're going to be giving to customers.  Building it two separate ways doesn't do that.

Reply:
I think its valuable to build the same way, but the on-host unpacking and setup will still be different. For example, the working directory for the tests here will be `\go\src\...\amazon-ecs-agent` instead of the normal runtime `\Amazon\ECS`.